tests/unittests/operators/target_view_tests.py

```
@@ -205,3 +203,144 @@ def test_label_description_override(self):
             )
         finally:
             ds.delete()
+
+
+class TestGroupedDatasetTargetView(unittest.TestCase):
+    """Resolution of ``ctx.target_view()`` for grouped datasets."""
+
+    dataset = None
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.20)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)